src/aws/osml/tile_server/viewpoint/database.py

```diff
@@ -43,7 +43,7 @@ class ViewpointStatusTable:
     A class used to represent the ViewpointStatusTable.
     """
 
-    def __init__(self, aws_ddb: ServiceResource) -> None:
+    def __init__(self, aws_ddb: ServiceResource, logger: Logger = getLogger(__name__)) -> None:
         """
         Initialize the ViewpointStatusTable and validate the table status.
 
```

Comment:
Update the docstring to reflect on the new parameters.

Reply:
fixed